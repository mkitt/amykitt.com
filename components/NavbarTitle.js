import React from 'react'
import css from '../styles/css'

const style = css({
  fontSize: 12,
  margin: '0 auto',
  textTransform: 'uppercase',
})

const bold = css({
  fontWeight: 700,
})

const normal = css({
  fontWeight: 400,
})

export default() => (
  <h1 className={style}>
    <span className={bold}>Hi. </span>
    <span className={normal}>
      If you like what you see below perhaps we can work together in the
      future.
    </span>
  </h1>
)
