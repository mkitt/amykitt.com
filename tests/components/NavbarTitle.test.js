import React from 'react'
import renderer from 'react-test-renderer'
import NavbarTitle from '../../components/NavbarTitle'

describe('NavbarTitle', () => {
  let component

  beforeEach(() => {
    component = (
      <NavbarTitle />
    )
  })

  it('shows the correct snapshot for the NavbarTitle component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
