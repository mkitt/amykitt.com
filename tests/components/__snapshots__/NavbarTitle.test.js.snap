// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarTitle shows the correct snapshot for the NavbarTitle component tree 1`] = `
<h1
  className={
    Object {
      "data-css-1a2lm7n": "",
    }
  }
>
  <span
    className={
      Object {
        "data-css-41sc57": "",
      }
    }
  >
    Hi. 
  </span>
  <span
    className={
      Object {
        "data-css-ucwin8": "",
      }
    }
  >
    If you like what you see below perhaps we can work together in the future.
  </span>
</h1>
`;
